src/lib/connectors/system/workflow-supplementary-billing.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const config = require('../../../../config')
+const { serviceRequest } = require('@envage/water-abstraction-helpers')
+
+/**
+ * Posts to system repo to flag new charge versions for SROC supplementary billing
```

Comment:
Not sure if this comment is correct or a copy/paste error from `src/lib/connectors/system/charge-version-supplementary-billing.js`?

Reply:
Good spot, thank you